scm-webapp/src/main/java/sonia/scm/api/v2/resources/UserApiKeyResource.java

```diff
@@ -89,14 +89,14 @@ public ApiKeyResource(ApiKeyService apiKeyService, ApiKeyCollectionToDtoMapper a
       mediaType = VndMediaType.ERROR_TYPE,
       schema = @Schema(implementation = ErrorDto.class)
     ))
-  public HalRepresentation getForCurrentUser() {
-    return apiKeyCollectionMapper.map(apiKeyService.getKeys());
+  public HalRepresentation findAll(@PathParam("id") String id) {
+    return apiKeyCollectionMapper.map(apiKeyService.getKeys(id), id);
```

Comment:
Without even checking why you did this it feels awkward to provide `id` two times. I am sure this api could cleaner somehow.

Reply:
Is this fixed ?